docs/docs/adding-comments.md

```diff
@@ -9,6 +9,7 @@ There are many options out there for adding comment functionality, several of th
 - [Disqus](https://disqus.com)
 - [Commento](https://commento.io)
 - [Facebook Comments](https://www.npmjs.com/package/react-facebook)
+- [Fast Comments](https://fastcomments.com/) with [Gatsby examples](https://github.com/FastComments/fastcomments-gatsbyjs-example/blob/master/src/pages/index.tsx#L11).
```

Comment:
```suggestion
- [Fast Comments](https://fastcomments.com/)
```

Linking to a single line in a file is brittle and can easily get out of date. Since we only link to the websites for the other instances we should keep things consistent